### DISCUSSION TITLE
How to get Ochard sitemap to include pages which are taxonomies

### DISCUSSION THREAD

**Discussion by esmemelissa** [OrchardCMS/OrchardCore#12869] [Q&A]
I'm working on getting the Orchard auto sitemap set up for my website but 2 of our page types are set up as taxonomies and I've been unable to get any pages in the path for these to pull through to the sitemap even when I've selected them under 'sitemap content types'. Has anyone else experienced this / knows how I can get Orchard to include them? I'm assuming the fact that they are taxonomies is what the issue is as all other content types are pulling through fine. TIA

**Comment by jtkech** [comment]
From memory a given type should have at least an `AutoroutePart`.

**Reply by esmemelissa** [reply]
> From memory a given type should have at least an `AutoroutePart`.

The 2 page types that aren't pulling through both have an AutoroutePart, I'm not sure what else they're missing for them not to be pulling through?
